config/i18n-tasks.yml

```diff
@@ -0,0 +1,121 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
```

Comment:
Because an engine exports stuff in configuration to the underlying application.  That application may not want this stuff cluttering up the config path.   How about generating this into the test app using the `test_app_generator.rb`